tests/test_clog.py

```diff
@@ -91,6 +91,16 @@ def test_multi_day(self):
             content = self._open_and_remove(log_filename)
             assert content == complete_line
 
+    def test_cant_open_stream(self):
```

Comment:
I mean, if the point of your change is to show the output, I'd write a test to show the output